docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/utils/index.md

```diff
@@ -0,0 +1,233 @@
+---
+sidebar_label: "Utils"
+sidebar_position: 103
+title: "Utils Quickstart"
+---
+
+The `snowplow-utils` package allows you to create your own custom `snowplow_base_events_this_run` table using macros that generate the required SQL code for you, allowing you to incorporate whatever custom event types, contexts of Snowplow data. Using this package will allow you to leverage the incremental nature of the Snowplow packages, meaning you can more easily build data models using our other packages such as `snowplow-web` and `snowplow-mobile`, as well as build your own completely custom packages without having to do the initial heavy lifting yourself. This can, however, be a bit complicated to set up and so for that purpose we've create this quickstart page to guide you through this process.
+
+## Requirements
+
+In addition to [dbt](https://github.com/dbt-labs/dbt) being installed and an events dataset being available in your database:
+
+- A dataset of Snowplow events from one of the [Snowplow trackers](docs/collecting-data/)
+
+```mdx-code-block
+import DbtPrivs from "@site/docs/reusable/dbt-privs/_index.md"
+
+<DbtPrivs/>
+```
+
+```mdx-code-block
+import DbtPackageInstallation from "@site/docs/reusable/dbt-package-installation/_index.md"
+
+<DbtPackageInstallation/>
+```
+
```

Comment:
Can you update this reusable block to include a prompt that they need to do a `dbt deps` after they have added it, I know it says it in their docs that we link to but it's nice to remind people.